dev/build/windows/makecoq_mingw.sh

```diff
@@ -917,7 +916,8 @@ function make_camlp4 {
 function make_camlp5 {
   make_ocaml
   make_findlib
-  if build_prep http://camlp5.gforge.inria.fr/distrib/src camlp5-6.14 tgz 1 ; then
+
+  if build_prep https://github.com/camlp5/camlp5/archive rel705 tar.gz 1 ; then
```

Comment:
The file in the chache and the log files will ne names rel705 now.
You should add an optional module name like camlp5-rel705 after the last argument (the 1) if the file name is not a good module name.

Reply:
Done, thanks!